features/step_definitions/work_steps.rb

```diff
@@ -57,29 +56,30 @@
   unless relationship.blank?
     fill_in("work[relationship_string]", with: relationship)
   end
+  select(pseud, from: "work[author_attributes][ids][]") unless pseud.blank?
   screenshot_and_save_page
   fill_in("work_recipients", with: "#{recipient}") unless recipient.blank?
 end
 
 # This is the same regexp as above
-When /^I post (?:a|the) work "([^"]*)"(?: with fandom "([^"]*)")?(?: with character "([^"]*)")?(?: with second character "([^"]*)")?(?: with freeform "([^"]*)")?(?: with second freeform "([^"]*)")?(?: with category "([^"]*)")?(?: (?:in|to) (?:the )?collection "([^"]*)")?(?: as a gift (?:for|to) "([^"]*)")?(?: as part of a series "([^"]*)")?(?: with relationship "([^"]*)")?$/ do |title, fandom, character, character2, freeform, freeform2, category, collection, recipient, series, relationship|
+When /^I post (?:a|the) work "([^"]*)"(?: with fandom "([^"]*)")?(?: with character "([^"]*)")?(?: with second character "([^"]*)")?(?: with freeform "([^"]*)")?(?: with second freeform "([^"]*)")?(?: with category "([^"]*)")?(?: (?:in|to) (?:the )?collection "([^"]*)")?(?: as a gift (?:for|to) "([^"]*)")?(?: as part of a series "([^"]*)")?(?: with relationship "([^"]*)")?(?: using the pseud "([^"]*)")?$/ do |title, fandom, character, character2, freeform, freeform2, category, collection, recipient, series, relationship, pseud|
   # If the work is already a draft then visit the preview page and post it
   work = Work.find_by_title(title)
   if work
     visit preview_work_url(work)
     click_button("Post")
   else
     # Note: this will match the above regexp and work just fine even if all the options are blank!
-    step %{I set up the draft "#{title}" with fandom "#{fandom}" with character "#{character}" with second character "#{character2}" with freeform "#{freeform}" with second freeform "#{freeform2}" with category "#{category}" in collection "#{collection}" as a gift to "#{recipient}" as part of a series "#{series}" with relationship "#{relationship}"}
+    step %{I set up the draft "#{title}" with fandom "#{fandom}" with character "#{character}" with second character "#{character2}" with freeform "#{freeform}" with second freeform "#{freeform2}" with category "#{category}" in collection "#{collection}" as a gift to "#{recipient}" as part of a series "#{series}" with relationship "#{relationship}" using the pseud "#{pseud}"}
     click_button("Post Without Preview")
   end
   Work.tire.index.refresh
 end
 
-# Again, same regexp, it just creates a draft and not a posted 
+# Again, same regexp, it just creates a draft and not a posted
 # To test posting after preview, use: Given the draft "Foo"
 # Then use: When I post the work "Foo"
-# and the above step 
+# and the above step
```

Comment:
This is a bit of feature creep however.

The comment says "Again, same regexp, it just creates a draft and not a posted"

But the regex only takes "title, fandom, freeform, category, collection, recipient" while the other one takes "title, fandom, character, character2, freeform, freeform2, category, collection, recipient, series, relationship, pseud" 

Given that I was probably the last person to extend the regex I would be happy to do it however you are working here right now. Does that make sense ?